modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -1329,7 +1330,7 @@ public static bool HtmlIsValid(ReadOnlySpan<char> color)
         /// <returns>A string representation of this color.</returns>
         public readonly string ToString(string? format)
         {
-            return $"({R.ToString(format)}, {G.ToString(format)}, {B.ToString(format)}, {A.ToString(format)})";
+            return $"({R.ToString(format, null)}, {G.ToString(format, null)}, {B.ToString(format, null)}, {A.ToString(format, null)})";
```

Comment:
Ditto for `Plane`, `Projection`, `Quaternion`, `Vector2`, `Vector2I`, `Vector3`, `Vector3I`, `Vector4`, `Vector4I`.

Do we really want to use the current culture instead of invariant here? E.g. on my machine, this would represent floating points as `3,1415927`, it feels wrong compared to `3.1415927`. I think we should default to `CultureInfo.InvariantCulture.NumberFormat` instead.

Reply:
While I agree, this was to the end of keeping the previous behavior intact. Plus, we'd probably want a way to change the culture if we're changing the default, which would be outside the scope of this PR. I'm implementing something similar in a seperate PR via IFormattable (#83159), so this default could be applied there instead

Alternatively, if everyone is onboard with making it `CultureInfo.InvariantCulture.NumberFormat`, I'm completely fine with applying that instead. Just a matter of gauging the other contributors' thoughts